landscapes/ticuna/landscape.ini

```diff
@@ -0,0 +1,28 @@
+[landscape]
+
+name = Ticuna
+author = Priscila Faulhaber images combined into this 360-degree view.
+
+type = spherical
+
+maptex = ticuna.png
+
+
+angle_rotatez = 150.00
+
+[location]
+
+planet = Earth
+
+latitude = -3d47'00"
+
+longitude = -69d26'00"
+
+altitude = 20
+
+atmospheric_extinction_coefficient = 0.27
+
+atmospheric_temperature = 10.0
+
+atmospheric_pressure = 1013.0
+light_pollution = 7
```

Comment:
Are you sure about 7? This is "Suburban/urban transition". A small village without street lights may rather have Bortle index 3. See Appendix B of the Stellarium User Guide.

Location should also have country and timezone entries. See section 7.1.1. for more info.

Reply:
Change made